sample_factory/algo/runners/runner.py

```diff
@@ -559,6 +568,9 @@ def _check_heartbeat(self):
             log.error(f"Stopping training from lack of heartbeats from {', '.join(type_list)}")
             self._stop_training()
 
+        for p_name, qsize in self.queue_size_dict.items():
```

Comment:
Maybe say here that the process has <N> messages in the outgoing signal queue.

And let's only report this if qsize is bigger than some threshold, like 5.